examples/worker/offscreenCanvas.js

```diff
@@ -0,0 +1,56 @@
+self.importScripts( '../../build/three.js' );
+
+self.onmessage = function ( message ) {
+
+	var data = message.data;
+	init( data.drawingSurface, data.width, data.height, data.pixelRatio );
+
+};
+
+var camera, scene, renderer, mesh, clock;
+
+function init( offscreen, width, height, pixelRatio ) {
+
+	camera = new THREE.PerspectiveCamera( 70, width / height, 1, 1000 );
+	camera.position.z = 400;
+
+	scene = new THREE.Scene();
+
+	clock = new THREE.Clock();
+
+	var loader = new THREE.ImageBitmapLoader();
```

Comment:
I think it might be wise to add a comment here stating that other types of loaders that use the `Image` class or `document.createElement` can not be used in a worker.

Reply:
👍